### PR TITLE
DOC-10047 update service account roles for log export and audit logs (on cloud-2.0 branch)

### PR DESCRIPTION
**Pull request by florence-crl**
Fixes DOC-10047

In export-logs-advanced.md, listed the roles a service account needs.

### CODE REVIEW

File: src/current/cockroachcloud/export-logs-advanced.md [hunk 1]
```diff
@@ -19,17 +19,19 @@ To configure and manage log export for your CockroachDB {{ site.data.products.ad
 https://cockroachlabs.cloud/api/v1/clusters/{your_cluster_id}/logexport
 ~~~
 
-Access to the `logexport` endpoint requires a valid CockroachDB {{ site.data.products.cloud }} [service account]({% link cockroachcloud/managing-access.md %}#manage-service-accounts) with the appropriate permissions (`admin` privilege or Cluster Admin role).
+Access to the `logexport` endpoint requires a valid CockroachDB {{ site.data.products.cloud }} [service account]({% link cockroachcloud/managing-access.md %}#manage-service-accounts) assigned one of the following [roles]({% link cockroachcloud/managing-access.md %}#edit-roles-on-a-service-account):
 
-The following methods are available for use with the `logexport` endpoint, and require the listed service account permissions:
+- [Organization Admin]({% link cockroachcloud/authorization.md %}#org-administrator)
```
Comment (taroface) [inline src/current/cockroachcloud/export-logs-advanced.md:24]: This is the wording in our docs, unless it changed?

```suggestion
- [Org Administrator]({% link cockroachcloud/authorization.md %}#org-administrator)
```

Comment (florence-crl) [reply src/current/cockroachcloud/export-logs-advanced.md:24]: @Amruta-Ranade ,  is it ok I modify the [authorization page](https://deploy-preview-18456--cockroachdb-docs.netlify.app/docs/cockroachcloud/authorization#org-administrator) to use the correct role name -`Organization Admin`? That is what I am seeing in production: 
Screenshot 2024-04-10 at 3 19 52 PM

Comment (Amruta-Ranade) [reply src/current/cockroachcloud/export-logs-advanced.md:24]: Will defer to @mdlinville since it's security-related

Comment (florence-crl) [reply src/current/cockroachcloud/export-logs-advanced.md:24]: I discussed with @mdlinville that I will modify this PR to match the existing docs. See [this slack thread](https://cockroachlabs.slack.com/archives/C01CD39JX8E/p1712849732265149) for discussion about updating the role name in bulk in another PR.

**Review by taroface — APPROVED** [review]

**Pull request merged**